packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionHistoryTable.tsx

```diff
@@ -133,230 +196,128 @@ export const TransactionHistoryTable = ({
+            <HistoryLoader />
           </div>
         ) : (
-          <div className="flex justify-between">
-            <div className="flex justify-start space-x-1">
-              <FailedChainPairsTooltip />
-              <span className="text-sm">
+          <div className="flex items-center justify-between">
+            <div className="flex items-center justify-start space-x-1">
+              <FailedChainPairsTooltip failedChainPairs={failedChainPairs} />
+              <span className="text-xs">
                 Showing {transactions.length}{' '}
                 {isPendingTab ? 'pending' : 'settled'} transactions made in{' '}
                 {oldestTxTimeAgoString}.
               </span>
             </div>
 
-            {!completed && (
-              <button
-                onClick={resume}
-                className="arb-hover text-sm"
-                aria-label="Load more"
-              >
-                <div className="flex space-x-1 rounded border border-black px-2 py-1">
-                  <span>Load more</span>
-                  <ArrowDownOnSquareIcon width={16} />
-                </div>
-              </button>
-            )}
+            {!completed && <LoadMoreButton onClick={resume} />}
           </div>
         )}
         <div>{pendingTokenDepositsCount > 0 && <PendingDepositWarning />}</div>
       </div>
+
       <AutoSizer disableHeight>
         {() => (
           <Table
             width={960}
             height={tableHeight}
-            rowHeight={({ index }) => getRowHeight(index)}
+            rowHeight={60}
             rowCount={transactions.length}
             headerHeight={52}
             headerRowRenderer={props => (
-              <div className="flex w-[960px] bg-white">{props.columns}</div>
+              <div className="mx-4 flex w-[920px] border-b border-white/30 text-white">
+                {props.columns}
+              </div>
             )}
-            className="table-auto"
+            className="table-auto last:border-b-0"
             rowGetter={({ index }) => transactions[index]}
             rowRenderer={({ index, style }) => {
               const tx = transactions[index]
-              const isEvenRow = index % 2 === 0
 
               if (!tx) {
                 return null
               }
 
-              const bgColorSettled = isEvenRow ? 'bg-cyan' : 'bg-white'
-              const bgColorPending = 'bg-orange'
-
-              const bgColor = isPending(tx) ? bgColorPending : bgColorSettled
-
+              const isLastRow = index + 1 === transactions.length
               const key = `${tx.parentChainId}-${tx.childChainId}-${tx.txId}`
+              const secondsPassed = dayjs().diff(dayjs(tx.createdAt), 'second')
+
+              // only blink the topmost tx, in case many txs are queued in a short amount of time
+              const isTopmostPendingTx =
+                transactions.filter(isTxPending)[0]?.txId === tx.txId
```

Comment:
Can we memo the transaction id for the top most pending id maybe? So we don't need to run this every for every line